<launch>
    <arg name="rviz_config" default="cam_pose_opt"/>
    <arg name="rviz" default="true"/>

    <param name="use_sim_time" value="false"/>

    <node name="camera_pose_optimization" pkg="frontier_exploration" type="camera_pose_optimization.py" output="screen"/>

    <node if="$(arg rviz)" name="rviz" pkg="rviz" type="rviz" args="-d $(find frontier_exploration)/config/$(arg rviz_config).rviz"/>
</launch>
